### ISSUE TITLE
No extensions support in ariadne.contrib.django.views.GraphQLView

### ISSUE THREAD

**Issue by markedwards** [mirumee/ariadne#362]
As far as I can tell, in order to enable an extension using the Django integration, I would pass an extensions argument to `GraphQLView.as_view()`.  However, that results in a Django error:

```
GraphQLView() received an invalid keyword 'extensions'. as_view only accepts arguments that are already attributes of the class.
```

Is this just a missing feature of GraphQLView at the moment?

**Comment by rafalp** [comment]
There's PR for that, but its missing tests: https://github.com/mirumee/ariadne/pull/284

**Comment by markedwards** [comment]
@rafalp wow, that was a fast response :-D.  I'll see if I can push that PR along, thanks!

**Comment by rafalp** [comment]
Merged #284
